scrapers/RealityKings.yml

```diff
@@ -66,19 +66,29 @@ xPathScrapers:
             with: https://www.realitykings.com
 
   sceneScraper:
+    common:
+      $section: //img[@alt=""][contains(@src,"poster")]/../../.. #img>div>div>section
     scene:
-      Title: //h2[contains(@class, "hzrLcs")]/text()
+      Title: $section//h1/text()|$section//h2/text()
       Date:
-        selector: //div[contains(@class, "jtRRdJ")]/text()
+        selector: //div[text()="Release Date:"]/../text()
         parseDate: January 2, 2006
-      Details: //p[contains(@class, "kLbWwG")]/text()
+      Details: //div[text()="Description:"]/../div[not(@class)]/text()
       Tags:
-        Name: //div[contains(@class, "cepogm")]/a/text()
+        Name: 
+            selector: //div[text()="Categories:"]/../a/text()[1]
       Performers:
-        Name: //div[contains(@class, "dEgjPa")]/span/a/text()
+        Name: $section//a[@href and @title]/text()
       Studio:
-        Name: //a[contains(@class, "kXGSit")]/text()
-      Image: //img[contains(@class, "lSdde")]/@src
+        Name: 
+            selector: $section//a[contains(@href,"site=")]/@title|//link[@rel="canonical"]/@href
+            postProcess:
+                - replace: 
+                    - regex: .*www\.
+                      with: ""
+                    - regex: \.com.+
+                      with: ""
```

Comment:
```yaml
Name: //a[contains(@href,"/scenes?site")]/text()
```
This is what I was looking at for the studio. It picks up the studio name with spaces vs parsing the URL. Not sure if there is a best practice for this: having the studio name with spaces vs no spaces.

Reply:
the specific selector `Name: //a[contains(@href,"/scenes?site")]/text()`  doesnt work with some sites eg `twistysnetwork` as some divs are present in the anchor text or sweetsinner where it doesn't match at all